MagentoEcsFargate/.gitignore

```diff
@@ -0,0 +1,10 @@
+*.swp
```

Comment:
Can we also ignore `.DS_Store` here? (and remove it from this PR.

Reply:
Sure